.github/DISCUSSION_TEMPLATE/c-discord-to-minecraft-chat-forwarding.yml

```
@@ -0,0 +1,40 @@
+title: "[DC -> MC] "
+labels: ["waiting for support"]
+body:
+- type: markdown
+  attributes:
+    value: |
+      __Ensure that the Message Content Intent is enabled__
+
+      **Even if you think this is done, refresh the page and make sure it saved correctly!** If messages are only being passed when they reply to/mention the bot then this is the solution.
+
+      1. Go to https://discord.com/developers/applications
+      2. Choose your bot (the bot is only visible to the Discord user that created it!)
+      3. Go to the "Bot" tab on the left
+      4. Check that the "MESSAGE CONTENT INTENT" is enabled (you may need to scroll down to see the option)
+          Note: After enabling the option you will need to restart your server
+- type: input
```

```suggestion
          Note: After enabling the option you will need to restart your server

      ---
- type: checkboxes
```

- Improves visibility
- Wrong input type was used